src/main/java/com/minecolonies/coremod/colony/jobs/JobEnchanter.java

```diff
@@ -0,0 +1,135 @@
+package com.minecolonies.coremod.colony.jobs;
+
+import com.minecolonies.api.client.render.modeltype.BipedModelType;
+import com.minecolonies.api.colony.ICitizenData;
+import com.minecolonies.api.colony.jobs.ModJobs;
+import com.minecolonies.api.colony.jobs.registry.JobEntry;
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.coremod.entity.ai.citizen.enchanter.EntityAIWorkEnchanter;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+public class JobEnchanter extends AbstractJob
+{
+    /**
+     * The tag to store the building he is currently draining from.
+     */
+    private static final String TAG_BUILDING_TO_DRAIN = "buildingtodrain";
+
+    /**
+     * The waiting ticks currently.
+     */
+    private static final String TAG_WAITING_TICKS = "waitingticks";
+
+    /**
+     * Max waiting ticks.
+     */
+    private static final int MAX_WAITING_TICKS = 60;
+
+    /**
+     * Pos of the worker to drain from.
+     */
+    private BlockPos posToDrainFrom = null;
+
+    /**
+     * The waiting ticks of the enchanter at the current pos.
+     */
+    private int waitingTicks = 0;
+
+    /**
+     * Initialize citizen data.
+     *
+     * @param entity the citizen data.
+     */
+    public JobEnchanter(final ICitizenData entity)
+    {
+        super(entity);
+    }
+
+    @Override
+    public JobEntry getJobRegistryEntry()
+    {
+        return ModJobs.enchanter;
+    }
+
+    @Override
+    public String getName()
+    {
+        return "com.minecolonies.coremod.job.enchanter";
+    }
+
+    /**
+     * Get the RenderBipedCitizen.Model to use when the Citizen performs this job role.
+     *
+     * @return Model of the citizen.
+     */
+    @NotNull
+    @Override
+    public BipedModelType getModel()
+    {
+        return BipedModelType.STUDENT;
```

Comment:
I take it this is temporary until we have a nice model to use?

Reply:
yes